R/ckan_fetch.R

```diff
@@ -59,15 +60,15 @@
 #' plot(x[, c("mun_name", "geometry")])
 #'
 #' }
-ckan_fetch <- function(x, store = "session", path = "file", format = NULL, ...) {
+ckan_fetch <- function(x, store = "session", path = "file", format = NULL, key = get_default_key(), ...) {
   store <- match.arg(store, c("session", "disk"))
   file_fmt <- file_fmt(x)
   if (identical(file_fmt, character(0)) & is.null(format)) {
     stop("File format is not available from URL; please specify via `format` argument.")
   }
   fmt <- ifelse(identical(file_fmt, character(0)), format, file_fmt)
   fmt <- tolower(fmt)
-  res <- fetch_GET(x, store, path, format = fmt, ...)
+  res <- fetch_GET(x, store, path, format = fmt, key = get_default_key(), ...)
```

Comment:
since the user can pass a `key` value into the fxn call, we need to capture that here - i.e., `key = key`

Reply:
its fine if they set their key outside of the fxn call but if they pass the key to the fxn call then it won't be passed on to `fetch_GET`

Reply:
fixed, thanks!